packages/flutter/lib/src/widgets/form.dart

```diff
@@ -293,19 +293,45 @@ class FormState extends State<Form> {
   /// returns true if there are no errors.
   ///
   /// The form will rebuild to report the results.
+  /// See also: [validateGranually]
```

Comment:
nit: this should be formatted like other `See also:` in the framework.

For example from `SelectableRegion`:

```dart
/// See also:
///  * [SelectableRegion], which provides an overview of the selection system.
```